### DISCUSSION TITLE
Bug: Make Archaeologist list count for ArchaeologistTableHeaders reflect correct count.

### DISCUSSION THREAD

**Discussion by boob1e** [sarcophagus-org/sarcophagus-v2-specs#19] [Ideas]
Currently, if the isFreeBondGreater flag is false, we hide the rows from the user so they cannot make a selection for an archaeologist that does not have enough funds to handle both the bounty and the storage fee. 

But, you still see the count of hidden items in the table header. This is confusing:
![Screen Shot 2022-05-01 at 11 52 21 AM](https://user-images.githubusercontent.com/9342600/166160806-4c3121b0-bdec-4232-95a1-f13aa552aa22.png)

To fix bug would probably involve moving the isFreeBondGreater calculation up into parent and do the calculation on the list within index.tsx. Then we can have the subset of items that have isFreeBondGreater true filtered out for the count being fed into header.

**Comment by adamgall** [comment]
Thank you for the bug report @raytorres280!

Can you please move (or copy) this bug report to the [`sarcophagus-app`](https://github.com/sarcophagus-org/sarcophagus-app) repository?

Since this bug comes directly from that codebase, it'll be good to track it there.

**Reply by boob1e** [reply]
my bad. that makes more sense doesn't it

**Comment by boob1e** [comment]
not letting me transfer discussion to other repo. no repos in transfer search results.
